Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+ADD ca-certificates.crt /etc/ssl/certs/
```

Comment:
I believe you can just use the https url here and save yourself the extra makefile task to pull it down.